skycatalogs/__init__.py

```diff
@@ -1,5 +1,3 @@
 from ._version import __version__
 
 from .skyCatalogs import *
-from .main_catalog_creator import *
-from .flux_catalog_creator import *
```

Comment:
This is technically a breaking change, but given that we only recently had a major bump and it's unlikely that users rely on these being imported automatically, I think it's okay to include it in the next release.

Reply:
I agree those imports don't belong there.  The catalog creation code depends on skyCatalogs.py but not the other way around - at least it shouldn't.